scripts/tcs_startup.sh

```diff
@@ -14,66 +14,73 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-enclave_manager="${TCF_HOME}/examples/enclave_manager/tcf_enclave_manager/enclave_manager.py"
-listener="${TCF_HOME}/common/python/connectors/direct/tcs_listener/tcs_listener.py"
-version="$(cat ${TCF_HOME}/VERSION)"
+ENCLAVE_MANAGER="${TCF_HOME}/examples/enclave_manager/tcf_enclave_manager/enclave_manager.py"
+LISTENER="${TCF_HOME}/common/python/connectors/direct/tcs_listener/tcs_listener.py"
+VERSION="$(cat ${TCF_HOME}/VERSION)"
+LMDB_URL="http://localhost:9090"
 
 # Trap handler
-trap 'stop_tcs_components' HUP INT QUIT ABRT ALRM TERM
+trap 'stop_avalon_components' HUP INT QUIT ABRT ALRM TERM
 
-start_tcs_components()
+start_avalon_components()
 {
     # Read Listener port from config file
-    listener_port=`grep listener_port ${TCF_HOME}/config/tcs_config.toml \
+    LISTENER_PORT=`grep listener_port ${TCF_HOME}/config/tcs_config.toml \
         | awk {'print $3'}`
-    if [ -z "$listener_port" ]; then
-        listener_port=1947
+    if [ -z "$LISTENER_PORT" ]; then
+        LISTENER_PORT=1947
     fi
 
-    echo "Starting Avalon Enclave Manager $version ..."
-    python3 $enclave_manager --lmdb_url http://avalon-lmdb:9090 &
```

Comment:
Also we need to document to start the kv_storage service has to be started first.

Reply:
Ok